nebula-logger/core/main/logger-engine/lwc/logger/__tests__/logger.test.js

```diff
@@ -1,6 +1,7 @@
 import { createElement } from 'lwc';
 import FORM_FACTOR from '@salesforce/client/formFactor';
-import { BrowserContext, enableSystemMessages, disableSystemMessages } from '../loggerService';
+import { BrowserContext } from '../loggerService';
+import LoggerService from '../loggerService';
```

Comment:
@jamessimone random readability/style-preference question for you - for imports like this (with a default + named exports), do you go for a one-liner? Or do you like to keep them separated?

```suggestion
import LoggerService, { BrowserContext } from '../loggerService';
```